packages/react-core/src/demos/examples/AlertGroup/AlertGroupDemo.tsx

```diff
@@ -0,0 +1,423 @@
+import React, { useEffect } from 'react';
+import {
+  Dropdown,
+  DropdownItem,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  KebabToggle,
+  NotificationBadge,
+  NotificationDrawer,
+  NotificationDrawerBody,
+  NotificationDrawerHeader,
+  NotificationDrawerList,
+  NotificationDrawerListItem,
+  NotificationDrawerListItemBody,
+  NotificationDrawerListItemHeader,
+  PageSection,
+  PageSectionVariants,
+  TextContent,
+  Text,
+  Title,
+  DropdownPosition,
+  EmptyStateVariant,
+  NumberInput,
+  Alert,
+  AlertProps,
+  AlertGroup,
+  AlertActionCloseButton
+} from '@patternfly/react-core';
+
+import BellIcon from '@patternfly/react-icons/dist/js/icons/bell-icon';
+import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon';
+import DashboardWrapper from '../DashboardWrapper';
+import DashboardHeader from '../DashboardHeader';
+
+interface NotificationProps {
+  title: string;
+  srTitle: string;
+  variant: 'default' | 'success' | 'danger' | 'warning' | 'info';
+  key: React.Key;
+  timestamp: string;
+  description: string;
+  isNotificationRead: boolean;
+}
+
+export const AlertGroupDemo: React.FunctionComponent = () => {
```

Comment:
In case we add more alert group demos down the line, maybe we can update this exported name (plus file name) to something along the lines of `AlertGroupToastWithNotificationDrawer`? What do you think @tompsota ?